runner/fixture.js

```diff
@@ -33,8 +33,14 @@ import { window, document } from "../src/globals";
 		}
 
 		var fixture = document.getElementById( "qunit-fixture" );
-		if ( fixture ) {
+		var resetFixtureType = typeof config.fixture;
+		if ( resetFixtureType === "string" ) {
+
+			// support user defined values for `config.fixture`
 			fixture.innerHTML = config.fixture;
```

Comment:
Seems like we would still want to replace the outerHTML as well, right? Otherwise attributes on `qunit-fixture` won't reset when using user-defined fixture markup.

Reply:
Great point, I'll add a test case for that and fix it up.

Reply:
Updated to address (with tests)...